File: python/exercicio1.py
class Product:
    def __init__(self, name, price, quantity):
        self.name = name
        self.price = price
        self.quantity = quantity

    def total(self):
        return self.price * self.quantity

    def update_price(self, percentage):
        self.price = self.price * (1 + percentage / 100)

    def __str__(self):
        return f"name: {self.name}, price: {self.price}, quantity: {self.quantity}"


p1 = Product("Laptop", 1000.0, 2)
print(p1)

File: javascript/exercicio1.js
class Product {
    constructor(name, price, quantity) {
        this.name = name;
        this.price = price;
        this.quantity = quantity;
    }

    total() {
        return this.price * this.quantity
    }

    updatePrice(percentage) {
        this.price = this.price * (1 + percentage / 100)
    }
}

const p1 = new Product("Laptop", 1000.0, 5)
const p2 = new Product("Headphones", 200.0, 2)

console.log()

